src/components/Synthetics/ExecutionPriceRow.tsx

```diff
@@ -0,0 +1,164 @@
+import { useMemo } from "react";
+import { t, Trans } from "@lingui/macro";
+import ExchangeInfoRow from "components/Exchange/ExchangeInfoRow";
+import StatsTooltipRow from "components/StatsTooltip/StatsTooltipRow";
+import TooltipWithPortal from "components/Tooltip/TooltipWithPortal";
+import ExternalLink from "components/ExternalLink/ExternalLink";
+import { OrderType } from "domain/synthetics/orders/types";
+import { formatDeltaUsd, formatUsd } from "lib/numbers";
+import { getPositiveOrNegativeClass } from "lib/utils";
+import { formatAcceptablePrice } from "domain/synthetics/positions";
+import { bigMath } from "lib/bigmath";
+import { formatPercentage } from "lib/numbers";
+import { TradeFlags, TriggerThresholdType } from "domain/synthetics/trade";
+import { TradeFees } from "domain/synthetics/trade";
+
+interface Props {
+  tradeFlags: TradeFlags;
+  displayDecimals?: number;
+  fees?: TradeFees;
+  executionPrice?: bigint;
+  acceptablePrice?: bigint;
+  triggerOrderType?: OrderType.LimitDecrease | OrderType.StopLossDecrease;
+}
+
+export function ExecutionPriceRow({
```

Comment:
```js
export const ExecutionPriceRow = memo(({
...
}: Props) => {
...
});
```

Reply:
I'd like to know your thoughts on this syntax. I believe it's better because we use a named function for the component instead of an arrow func, which can make error tracing much more informative. But it also could be a bit confusing:

```
export const ExecutionPriceRow = memo(function ExecutionPriceRow({
  fees,
  executionPrice,
  tradeFlags,
  acceptablePrice,
  triggerOrderType,
  displayDecimals,
}: Props) {
}
```